pages-e2e/package.json

```diff
@@ -4,7 +4,7 @@
 	"scripts": {
 		"//test:e2e 1": "we start with npm i because this fixes an existing bug in the pages-e2e-test-runner-cli",
 		"//test:e2e 2": "which fails to detect test suites in the files after any run of the test-runner cli",
-		"test:e2e": "npm i ; npx @cfpreview/pages-e2e-test-runner-cli --environment=local"
+		"test:e2e": "npm i ; npx @cfpreview/pages-e2e-test-runner-cli  --install-wrangler=3.17.0 --environment=local"
```

Comment:
Is this basically moved from the GH Action above?
Is there no later version that we can use?

Reply:
Just not to get out of scope (/delay the PR) here I figured I wouldn't try to increase this to the latest in this PR

If it's ok I will look into it separately 🙂 

PS: I did briefly try with `latest`/`^3.28.0` but it seems to not work properly, I'll need to investigate that and potentially open an issue in this repo and/or workers-sdk

Reply:
No problem. Would be good to find out why it is not working with later versions. I think we introduced the new dev server proxy in 3.19?

Reply:
ok thanks, I'll have a look 😄